sub-projects/device/src/AWSClass.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import json
+
+#this class is for telescreen_device activity
+class AWS(object):
+	
+	#initalize the session for AWS
+	def __init__(self):
+		self.session = boto3.Session(profile_name='default')
+	
+	#uploading img to S3
+	def uploadS3(self, img):
+		s3 = self.session.resource('s3')
+		s3.meta.client.upload_file(img, 'checkfaces', img)
+		
```

Comment:
Class methods should be separated by only a single blank line.

See https://google.github.io/styleguide/pyguide.html#Blank_Lines